modular_skyrat/modules/advanced_shuttles/code/shuttles.dm

```diff
@@ -128,6 +128,16 @@
 /datum/map_template/shuttle/cargo/skyrat/delta
 	suffix = "delta_skyrat"	//I hate this. Delta station is one tile different docking-wise, which fucks it ALL up unless we either a) change the map (this would be nonmodular and also press the engine against disposals) or b) this (actually easy, just dumb)
 
+/datum/map_template/shuttle/cargo/skyrat/delta
```

Comment:
you messed with pathes, probably? You redefined Delta's shuttle and that's why you getting errors for Deltastation and Raptor

Reply:
my ass straight up forgot to change the path name 😭 thanks for pointing this out lmao